spec/unit/recipes/default_spec.rb

```diff
@@ -0,0 +1,62 @@
+require 'spec_helper'
+
+describe 'yum-almalinux::default' do
+  step_into :yum_alma_baseos
+  step_into :yum_alma_appstream
+  step_into :yum_alma_extras
+
+  %w(8).each do |v|
```

Comment:
I'll work on adding 9 to fauxhai-ng but won't be included for a while. You should add a `TODO` comment here so we can add that once it's done.